Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.cs

```diff
@@ -45,7 +45,7 @@ private static void Textbox_Loaded(object sender, RoutedEventArgs e)
         {
             var textbox = (TextBox)sender;
 
-            // incase no value is provided us it as normal textbox
+            // in case no value is provided us it as normal textbox
```

Comment:
Missing initial uppercase, also I don't understand what this comment means.
Should it be "provided to us", or was it meant to be "provided, use it..." or something else?